docs/pages/identity-governance/aws-iam-identity-center/guide.mdx

```diff
@@ -141,10 +152,91 @@ It may take a few minutes for the initial sync to complete.
 Imported Access Lists should show the same members as their corresponding
 Identity Center groups.
 
-## Step 6/6. Connect to AWS
+## Step 6/7. Configure access
+
+You need two kinds of Teleport role permissions to access the AWS accounts:
+- A permission that allows access to the SAML service provider which logs you into the 
+  AWS Console. You configured this SAML service provider for the AWS IAM Identity Center
+  in Step 3.
+- A permission that allows access to the AWS account and permission set. 
+
+### Configure access to the SAML service provider
+
+First, we will configure an access to the SAML service provider that logs you 
+into the AWS Console.
+
+In the Teleport Web UI, from side navigation menu, select "Zero Trust Access > Roles".
+From the "Roles" UI, click "Create New Role" button. Switch to the YAML editor.
+![Teleport role editor YAML mode](../../../img/igs/entraid/entra-teleport-role-yaml-editor.png)
+
+Copy the role spec shown below and paste it in the role editor to create a new role.
+```yaml
+kind: role
+version: v8
+metadata:
+  name: identity-center-access
+spec:
+  allow:
+    app_labels:
+      'teleport.dev/origin': 'aws-identity-center'
+```
+
+Notice that in the role spec above, the `app_labels` defines a `'teleport.dev/origin': 'aws-identity-center'`
+label. This is the default resource label which Teleport assigns to the SAML service 
+provider resource created for the AWS IAM Identity Center. 
+
+Grant this role to the users who need access to the AWS console. 
```

Comment:
This role alone wouldn't really give them access to anything would it? They need accounts and permission sets too. I would just keep the section below where you show the full role spec, and explain there that the app label gives access to the SAML provider. Otherwise this example standalone is confusing.